hw5-3/main.c

```diff
@@ -0,0 +1,85 @@
+#include <stdio.h>
+#include "stack.h"
+
+int isSndLessPriority(char operator1, char operator2) {
+    if (((operator2 == '+') || (operator2 == '-')) && ((operator1 == '*') || (operator1 == '/'))) {
+        return 1;
+    }
+    return 0;
+}
```

Comment:
```suggestion
bool isSndLessPriority(char operator1, char operator2) {
    return (operator2 == '+' || operator2 == '-') && 
           (operator1 == '*' || operator1 == '/');
}
```
Условный оператор тут не нужен. Сразу возвращай результат условия.
И скобки можно подчистить